server/apps/main/templates/main/about_us.html

```diff
@@ -124,7 +128,22 @@ <h5 class="text-info" id="data-usage"><em></em>Data Usage</h5>
           <p>You can also change your mind at any time by going to My-Experiences and selecting which options you would
             prefer.</p>
         </div>
-      </div>
+        <div class="use-terms">
+          <h5 class="text-info" id="use-terms"><em></em>Terms of Use</h5>
+          <p> AutSPACEs is realeased under an open MIT license. This means that the code is freely available to reuse and modify for any purpose, as long as they credit the AutSPACEs community.</p>   
+          <p> The data we collect for researcher is only available by application, and researchers must commit to following our values.</p>
+        </div>
+          <h5 class="text-info" id="our-values"><em></em>Our Values</h5>
+          <p><b>"Nothing About Us Without Us"</b></p>
+          <p>We believe that the people who will be affected most by research should have opportunities to make decisions, set the direction, and be involved in the research process throughout.</p>
+          <p><b>Allied to autistic people</b></p>
+          <p>We believe that the world should work for autistic people so that they can flourish as well as function, and we share a mission with our original sponsor Autistica for autistic people have happier, healthier, longer lives. Autism research should fundamentally serve autistic people and improve their lives.</p>
```

Comment:
```suggestion
          <p>We believe that the world needs to work for autistic people so that they can flourish as well as function. We share a mission with our original sponsor Autistica for autistic people have happier, healthier, longer lives. Autism research should fundamentally serve autistic people and improve their lives.</p>
```